src/sunburst.js

```diff
@@ -23,6 +20,8 @@ export default Kapsule({
     sort: { onChange(_, state) { state.needsReparse = true }},
     label: { default: d => d.name },
     labelOrientation: { default: 'auto' }, // angular, radial, auto
+    textSize: {default: 12},
```

Comment:
Even though this is already an improvement, we could go a step further and turn these props into accessor functions, just like most of the other props. That way you could set different font sizes / stroke widths per individual element.